src/DIRAC/Core/Base/Script.py

```diff
@@ -109,8 +109,20 @@ def registerSwitch(showKey, longKey, helpString, callback=False):
   localCfg.registerCmdOpt(showKey, longKey, helpString, callback)
 
 
-def getPositionalArgs():
-  return localCfg.getPositionalArguments()
+def registerArgument(description, mandatory=True, acceptedValues=None, default=None):
+  """ Register a new command line argument
+
```

Comment:
Point to the registerCmdArg function. Something like (I think)
```suggestion

See :func:`~DIRAC.ConfigurationSystem.Client.LocalConfiguration.LocalConfiguration.registerCmdArg` for the syntax of the ``description`` parameter.

```